addons/web/static/tests/webclient/actions/embedded_action.test.js

```diff
@@ -332,6 +335,60 @@ test("a view coming from a embedded can be saved in the embedded actions", async
     });
 });
 
+test("a view coming from a embedded with python_method can be saved in the embedded actions", async () => {
```

Comment:
There is a lot of code duplication between this test and the one above it - I don't necessarily see a clear way of doing so. Combining them into one would make it quite big and confusing which is not desirable. Anyways, just food for thought

Reply:
Yep, indeed, I agree with you. 🤔 

TBH, I did not really pay to attention to refactor or merge the tests to not loose the readability. Moreover, we plan to remove `python_method` field in embedded action model because the python method used in the embedded actions could be replaced by a server action.

See <https://www.odoo.com/odoo/project/4105/tasks/4184991>

So, I guess it is okay to keep that duplication only in that version. 🤷🏼‍♂️ 

What do you think? :)

Reply:
Sure then it's okay - if it's only one version being affected then it is not a big problem